### DISCUSSION TITLE
Homematic Wired-classic components are not recognized

### DISCUSSION THREAD

**Discussion by tgw-de** [SukramJ/aiohomematic#1471] [Q&A]
### The problem

Hello,
I am quite new to Home-Assist and installed the Homematic-local two days ago. During the installation I also passed the point that Wired is activated. 
However, no devices are recognized on the wired (not IP). All ports are open.
Where can I search or look something up? 

Klaus

### What version of HomematicIP (local) has the issue?

1.58.0

### What type of installation are you running?

Home Assistant OS

### What type of installation are you running for your homematic backend?

CCU3

### Which version of your homematic backend are you running?

3.75.6

### What hardware are you running for your system?

VMware-Image

### Which config details do you use

- [ ] TLS
- [ ] callback data ([see](https://github.com/danielperna84/custom_homematic#callback_host-and-callback_port))

### Which interfaces do you use?

- [X] Homematic IP
- [X] Homematic (Bidcos-RF)
- [ ] Groups (Heating-Group)
- [X] BidCos-Wired (HM-Wired)

**Comment by SukramJ** [comment]
> Where can I search or look something up?

Der erste Ort für die Fehlersuche ist immer die HA Log Datei `Einstellungen -> System -> Protokolle (vollständige Protokolle laden)`

**Comment by tgw-de** [comment]
Hallo und danke für die Antwort.
Ich lann da nicht wirklich etwas entdecken, ausser, dass die custom integrations angenörgelt werden. 
Kannst Du etwas entdecken?
[home-assistant_2024-04-02T14-57-41.917Z.log.zip](https://github.com/danielperna84/hahomematic/files/14838869/home-assistant_2024-04-02T14-57-41.917Z.log.zip)

**Reply by SukramJ** [reply]
Da ist nichts drin. 
Was für wired Gerätetypen verwendest Du?
Bitte häng mal die integration diagnostics an. (Rechte Maustaste auf die Integration. … herunterladen)

**Comment by tgw-de** [comment]
Ich benutze normale Classic Komponenten über das HM Gateway. Kein IP.
HMW-LC-Bl1-DR, HMW-LC-Sw2-DR, HMW-Sen-SC-12-DR, HMW-IO-12-Sw7-DR.
Die Diagnostics finde ich nicht. :( Rechter Mausknopf ist immer nur Browser-Erweiterungen.

**Reply by SukramJ** [reply]
Hast Du 3 Punkte rechts neben konfigurieren?

**Comment by tgw-de** [comment]
So hier bin ich wieder, sorry, dass ich mich nicht abgemeldet habe. Ich habs gefunden, Anbei die Datei.

[config_entry-homematicip_local-40caddf5f7d31dcbda8ad67acd90adff.zip](https://github.com/danielperna84/hahomematic/files/14850386/config_entry-homematicip_local-40caddf5f7d31dcbda8ad67acd90adff.zip)

**Reply by SukramJ** [reply]
Aktivier doch mal wired in der Konfiguration.

**Comment by tgw-de** [comment]
Bisher habe ich nirgends die Möglichkeit gefunden. Bei der Installation wurde ich das abgefragt und habe das auch sehr sicher eingeschaltet. Da hatte ich auch schon nach gesucht, hatte ich ja oben geschrieben. Nur wo kann ich es jetzt kontrollieren und aktivieren? Bei der Konfiguration wird es mir nicht angeboten.

**Reply by SukramJ** [reply]
Du klickst auf konfigurieren.

**Comment by tgw-de** [comment]
Da war ich schon ... falsch?
<img width="332" alt="2024-04-03 13_54_59" src="https://github.com/danielperna84/hahomematic/assets/52502868/ce385f94-cc9d-4bdb-be85-e3a946171218">

**Reply by Baxxy13** [reply]
"Absenden" klicken und auf der nächsten "Seite" unten "aktiviere Homematic-Wired (BidCos-Wired)" anhaken.

**Comment by tgw-de** [comment]
OK, danke Dir, ich hab an dem Punkt gebremst, weil ich mir nichts zerschießen wollte. Eventuell macht es Sinn, den Knopf "Absenden" in "Weiter" umzubenennen? Dann hätte ich mich getraut. :)

**Reply by SukramJ** [reply]
Mein Plan war ein paar Stunden zu warten, um zu sehen, ob Du selber mal auf den Button klickst.
Den ganzen Dialog kennst Du aus der Erstinstallation, so dass man das schon erwarten kann.

Ansonsten kann jedem natürlich die diversen Installationanleitungen-/Videos bzw. die Dokumentation empfehlen.
Bei den ganzen Smarthome Projekten ist Eigenständigkeit sehr willkommen, den auch hier gilt: Aus Fehlern lernt man.
